### PR TITLE
CLDR-15424 ansible: update upgrade playbook

### PR DESCRIPTION
**Pull request by srl295**
CLDR-15424

- [x] This PR completes the ticket.

### CODE REVIEW

File: tools/scripts/ansible/upgrade-playbook.yml [hunk 1]
```diff
@@ -1,7 +1,21 @@
 # A playbook for upgrading packages
 - hosts: all
+  vars_files:
+    - vars/main.yml
+    - vars/openliberty.yml
+    # - local-vars/local.yml
   become: yes
   tasks:
+  # TODO: should skip these
```
Comment (btangmu) [inline tools/scripts/ansible/upgrade-playbook.yml:9]: I don't understand this "TODO" comment -- if this PR finishes the ticket, why a TODO? What is it that should be skipped?

**Review by btangmu — CHANGES_REQUESTED** [review]

File: tools/scripts/ansible/upgrade-playbook.yml (hunk 1 above)
Comment (srl295) [reply tools/scripts/ansible/upgrade-playbook.yml:9]: it should only restart them if they are running.

**Comment by btangmu** [comment]
@srl any update on this? BRS "Upgrade server" points to CLDR-15176 which is related to this

**Comment by srl295** [comment]
> @SRL any update on this? BRS "Upgrade server" points to [CLDR-15176](https://unicode-org.atlassian.net/browse/CLDR-15176) which is related to this

This playbook is OK to use, it's just less than optimal because I think it will always restart the liberty server. Since we don't run this all the time, maybe that's OK. 

Do you want to try this change and if it is successful we can merge it?

File: tools/scripts/ansible/upgrade-playbook.yml (hunk 1 above)
Comment (btangmu) [reply tools/scripts/ansible/upgrade-playbook.yml:9]: looks like it's time for me to try this revised upgrade-playbook on smoketest

Comment (btangmu) [reply tools/scripts/ansible/upgrade-playbook.yml:9]: I already tested it last time (April 25, 2022)! -- old ticket https://unicode-org.atlassian.net/browse/CLDR-15581

**Comment by srl295** [comment]
We probably want to reboot after upgrade. So this might be ok

**Pull request merged**